go/tasks/plugins/presto/config/config.go

```diff
@@ -101,6 +102,7 @@ type Config struct {
 	Environment            config.URL           `json:"environment" pflag:",Environment endpoint for Presto to use"`
 	DefaultRoutingGroup    string               `json:"defaultRoutingGroup" pflag:",Default Presto routing group"`
 	DefaultUser            string               `json:"defaultUser" pflag:",Default Presto user"`
+	UseNamespaceAsUser     bool                 `json:"useNamespaceAsUser" pflag:",Use the K8s namespace as the user"`
```

Comment:
Do you see a world where we pass in the actual user principal as the user? for interactive usecases maybe?
should this be like:
```
UserParamSource []UserParamSource `json:"userParamSource" pflag:",An ordered list of sources to fallback to to fill in the user parameter when creating a presto job"
```

```
//go:generate enumer --type=UserParamSource --timprefix=UserParamSource --json

type UserParamSource int

const (
    UserParamSourceDefaultUser UserParamSource = iota
    UserParamSourceNamespace
    UserParamSourceEmpty
    UserParamSourceLoggedInUser
)
```

Reply:
sure why not.  will use this instead

Reply:
though I don't see the need to make this a list, can I just make it one thing for now?

Reply:
actually, this adds a fair bit of code, i'll add this if/when the need arises.